routers/repo/branch.go

```diff
@@ -110,6 +110,22 @@ func RestoreBranchPost(ctx *context.Context) {
 		return
 	}
 
+	// Don't return error below this
+	if err := repofiles.PushUpdate(
+		ctx.Repo.Repository,
+		branchName,
```

Comment:
Maybe we should use `deletedBranch.Name` here, as `branchName` from query seems only to be the fallback for the error message.

```suggestion
		deletedBranch.Name,
```